ui/src/app/app.constants.ts

```diff
@@ -84,6 +84,26 @@ export const Profile = {
   ExistingConnProfile: 'existing',
 }
 
+export const Dialect = {
+  PostgreSQLDialect: 'experimental_postgres'
```

Comment:
I think its better to have this as an enum and contain both `PostgreSQLDialect` and `GoogleSQLDialect` for future use.

Reply:
Added